src/gateway/kong-enterprise/index.md

```diff
@@ -11,7 +11,9 @@ It is the only solution that helps you accelerate your cloud journey by managing
 ## Enterprise Plugins
 
 Kong Enterprise offers access to 400+ out-of-box enterprise and community plugins. 
-It offers exclusive versions of OSS plugins like the [Rate-Limiting Advanced plugin](/hub/kong-inc/rate-limiting-advanced/) with added functionality such as the use of consumer groups, and database specific strategy. 
+It offers exclusive versions of OSS plugins like the [Rate Limiting Advanced plugin](/hub/kong-inc/rate-limiting-advanced/) with added functionality such as the use of consumer groups, and database specific strategy. It also provides Enterprise-exclusive functionality, such as authentication with 
+[OpenID Connect](/hub/kong-inc/openid-connect/), which lets you standardize identity provider (IdP) integrations.
```

Comment:
Just a thought as I was looking at this page. I think your solution works. I just noticed that as I was scanning this section, I looked at the list of plugin links in this section first and didn't see OIDC there. So people who are scanning might look at the list first.